src/components/common/Select.js

```diff
@@ -22,22 +25,80 @@ const Wrapper = styled.div`
   display: flex;
   align-items: center;
   position: relative;
+  height: 40px;
 `;
 
 const StyledDropdownArrow = styled(DropdownArrow)`
   position: absolute;
   right: 0.7em;
 `;
 
-const Select = ({ children, ...props }) => (
-  <Wrapper>
-    <StyledSelect {...props}>{children}</StyledSelect>
-    <StyledDropdownArrow />
-  </Wrapper>
-);
+const OptionsWrapper = styled.div`
+  position: absolute;
+  top: 50px;
+  left: 0;
+  width: 100%;
+  max-height: 160px;
+  overflow-y: scroll;
+  ${({ theme, isOpen }) => css`
+    background-color: ${theme.background.input};
+    border-radius: ${theme.border.radius};
+    border: 1px solid ${theme.background.input};
+    display: ${isOpen ? 'block' : 'none'};
+  `}
+`;
+
+const Option = styled.button`
+  display: block;
+  appearance: none;
+  height: 40px;
+  width: 100%;
+  text-align: initial;
+  border-width: 0;
+  padding: 0 0.7em;
```

Comment:
I think we usually use `rem` instead of `em` for relative measures, since`rem` depends on the root element and not the  father's size

Reply:
Makes sense, will fix this 👍